src/components/tag/Tag.constants.ts

```diff
@@ -0,0 +1,18 @@
+import { TTagVariant, TTagSizeToken } from './Tag.types';
+
+const TAG_VARIANTS: TTagVariant[] = [
+    'highlight',
+    'shortcut',
+    'general',
+    'info',
+    'warning',
+    'success',
+];
+
+const TAG_SIZES: TTagSizeToken[] = [25, 50, 75, 100, 200, 300];
+
+const DEFAULT_TAG_VARIANT: TTagVariant = 'general';
+
+const DEFAULT_TAG_SIZE: TTagSizeToken = 100;
+
+export { TAG_SIZES, TAG_VARIANTS, DEFAULT_TAG_VARIANT, DEFAULT_TAG_SIZE };
```

Comment:
nit: I tend to order them like this:

1. Collection of values
2. Default value for the collection
3. Collection of values
4. ...